concepts/pipeline/http_edge_spec.md

```diff
@@ -0,0 +1,105 @@
+# HTTP Edge Server Specification
+
+This document specifies the behavior of the server which is accepts Telemetry submissions from Firefox.
+
+The original implementation of the HTTP Edge Server was tracked in [Bug 1129222](https://bugzilla.mozilla.org/show_bug.cgi?id=1129222).
+
+## General Data Flow
+HTTP submissions come in from the wild, hit a load balancer, then optionally an nginx proxy, then the HTTP Edge Server described in this document. Data is accepted via a POST/PUT request from clients, which the server will wrap in a [Heka message](https://hekad.readthedocs.io/en/dev/message/index.html) and forward to two places: the [Services Data Pipeline](data_pipeline.md), where any further processing, analysis, and storage will be handled; as well as to a short-lived S3 bucket which will act as a fail-safe in case there is a processing error and/or data loss within the main Data Pipeline.
```

Comment:
The current Heka message link is here: http://mozilla-services.github.io/lua_sandbox/heka/message.html